module/applications/actor/group-sheet.mjs

```diff
@@ -1,433 +1,487 @@
+    },
+    actions: {
+      award: GroupActorSheet.#onAward,
+      editDescription: GroupActorSheet.#onEditDescription,
+      placeMembers: GroupActorSheet.#onPlaceMembers,
+      removeMember: GroupActorSheet.#onRemoveMember,
+      roll: GroupActorSheet.#onRoll
+    },
+    tab: "members"
+  };
+
+  /** @override */
+  static PARTS = {
+    header: {
+      template: "systems/dnd5e/templates/actors/group/header.hbs"
+    },
+    members: {
+      container: { classes: ["tab-body"], id: "tabs" },
+      template: "systems/dnd5e/templates/actors/group/members.hbs",
+      templates: ["systems/dnd5e/templates/actors/group/member.hbs"],
+      scrollable: [""]
+    },
+    inventory: {
+      container: { classes: ["tab-body"], id: "tabs" },
+      template: "systems/dnd5e/templates/actors/group/inventory.hbs",
+      templates: [
+        "systems/dnd5e/templates/inventory/inventory.hbs", "systems/dnd5e/templates/inventory/activity.hbs",
+        "systems/dnd5e/templates/inventory/containers.hbs"
+      ],
+      scrollable: [".sidebar", ".body"]
+    },
+    biography: {
+      container: { classes: ["tab-body"], id: "tabs" },
+      template: "systems/dnd5e/templates/actors/group/biography.hbs",
+      scrollable: [""]
+    },
+    tabs: {
+      id: "tabs",
+      classes: ["tabs-right"],
+      template: "systems/dnd5e/templates/shared/sidebar-tabs.hbs"
+    }
+  };
 
   /**
-   * IDs for items on the sheet that have been expanded.
-   * @type {Set<string>}
-   * @protected
+   * Application tabs.
+   * @type {SheetTabDescriptor5e}
    */
-  _expanded = new Set();
+  static TABS = [
+    { tab: "members", label: "DND5E.Group.Member.other", icon: "fa-solid fa-swords"},
+    { tab: "inventory", label: "DND5E.Inventory", svg: "systems/dnd5e/icons/svg/backpack.svg" },
+    { tab: "biography", label: "DND5E.Biography", icon: "fa-solid fa-feather" }
+  ];
 
   /* -------------------------------------------- */
-
-  /** @inheritDoc */
-  static get defaultOptions() {
-    return foundry.utils.mergeObject(super.defaultOptions, {
-      classes: ["dnd5e", "sheet", "actor", "group"],
-      template: "systems/dnd5e/templates/actors/group-sheet.hbs",
-      tabs: [{navSelector: ".tabs", contentSelector: ".sheet-body", initial: "members"}],
-      scrollY: ["dnd5e-inventory .inventory-list"],
-      width: 620,
-      height: 620,
-      elements: {
-        inventory: "dnd5e-inventory"
-      }
-    });
-  }
-
+  /*  Properties                                  */
   /* -------------------------------------------- */
 
   /**
-   * A set of item types that should be prevented from being dropped on this type of actor sheet.
-   * @type {Set<string>}
+   * Description currently being edited.
+   * @type {string|null}
    */
-  static unsupportedItemTypes = new Set(["background", "race", "class", "subclass", "feat"]);
+  editingDescriptionTarget = null;
+
+  /* -------------------------------------------- */
+
+  /** @override */
+  tabGroups = {
```

Comment:
The character and NPC sheets include this above with the static properties, but since it is an instance property it would probably make more sense to adjust those other two sheets to move this down to match this one.